collabora_online.views.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Implements hook_views_data_alter().
+ */
+function collabora_online_views_data_alter(array &$data): void {
+    $data['media']['collabora_preview'] =  [
+        'title' => t('Preview operation in collabora'),
+        'group' => t('Media'),
+        'field' =>  [
+            'title' => t('Media preview in collabora'),
```

Comment:
In my views UI the labels shows up as "Media: Media preview in collabora".
I don't know if this will always be the case.
It looks like other views field handler labels don't specify the entity type, unless it is different from the table they are attached to.

Reply:
Also, perhaps we should align with the labels in `collabora_online_entity_operation()`. So it would be "View in Collabora Online".

Reply:
Not sure about "View" vs "Preview" actually.

Reply:
The problem is we already have other places where it is called "View in Collabora Online".
Check `collabora_online_entity_operation()`.
The permission name is odd then, indeed.

Reply:
Then yes, I think labels show use same terms at least for the UI.

Since the only avaliable text for the user is this, not permissions label, I change links text to this.